File: c-programming/regresiveCounter.c
#include <stdio.h>

int regresiveCounter(){
  for(int i = 300;i>0;i--){
    printf("%d Seconds...\n",i);
  }
  return 0;
}

int main(){
  if(regresiveCounter() == 0){
    printf("The counter has endeed.\n");
  }
  return 0;
}

File: java-programming/regresiveCounter.java
class Main{
    public static int StartRegresiveCount(){
        for (int seconds = 300;seconds >0;seconds--){
            System.out.println(seconds+" Seconds..");
        }
        return 0;
    }
    public static void main(String[] args){

        int status = StartRegresiveCount();
        if(status == 0){
            System.out.println("The counter has endeed.");
        }
    }

}